### ISSUE TITLE
cannot delete the pictures I have uploaded

### ISSUE THREAD

**Issue by clear-train** [gradio-app/gradio#3893]
Hello, when I use the statement 
`f = gr.File(file_types=["image"], file_count="multiple", label="Input Images")` 
I find that I cannot delete the pictures I have uploaded, and the component box There is always orange-red flashing, what is the problem? I think it has been performing the upload process, resulting in the failure to perform the normal delete action.
Below is my code
``` python
import gradio as gr

def rank(files,prompt):
    images = [file.name for file in files]
    with torch.no_grad():
        ranking, rewards = model.inference_rank(prompt, images)
    return images, str(ranking), str(rewards)


with gr.Blocks() as demo:
    with gr.Row():
        with gr.Column():
            prompt = gr.inputs.Textbox(lines=2, label="Prompt")
            f = gr.File(file_types=["image"], file_count="multiple", label="Input Images")
            btn = gr.Button()
        with gr.Column():
            o = gr.Gallery().style(grid=4)
            ranking = gr.Textbox(label="Output Ranking")
            reward = gr.Textbox(label="Output Score")
    btn.click(rank,[f,prompt], [o,ranking,reward])

demo.launch()```

**Comment by abidlabs** [comment]
Hi @clear-train are you using the latest version of `gradio` (3.27)?

**Comment by freddyaboulton** [comment]
The orange-red flashing should be taken care of by #3859

**Comment by binary-husky** [comment]
#3853

**Comment by clear-train** [comment]
> Hi @clear-train are you using the latest version of (3.27)?`gradio`

nope my gradio version is 3.26.0 last night, but l've just upgraded it and the problem is still here.

**Comment by clear-train** [comment]
> #3853
thank you so much !

**Comment by abidlabs** [comment]
Sounds good, we just merged in #3859. Can you try testing by downloading `gradio` from `main`? You can install gradio from main by following the instructions here: https://gradio.app/docs/main

**Comment by clear-train** [comment]
thank you all, have a nice day!

**Comment by clear-train** [comment]
thank you all, have a nice day!
